### PR TITLE
feat: add allowEmptyValue parameter to sendCreateState

### LINKED ISSUE #25: [Feature] allowEmptyValue parameter in sendCreateState method
**Is your feature request related to a problem? Please describe.**
Currently there isn't any way of making a dynamicly created state empty.

**Describe the solution you'd like**
I would like to see a `allowEmptyValue` parameter added to the `sendCreateState` method with a overload just like the `stateUpdateMethod`

### PR DESCRIPTION
**Pull request by Pjiesco**
closes #25

### CODE REVIEW

File: Library/src/main/java/com/christophecvb/touchportal/TouchPortalPlugin.java [hunk 1]
```diff
@@ -626,19 +626,33 @@ public boolean sendStateUpdate(String stateId, Object value, boolean allowEmptyV
...
+    }
+
     /**
      * Send a Create a State Message to the Touch Portal Plugin System
      *
      * @param categoryId  String
      * @param stateId     String
      * @param description String
      * @param value       Object
+     * @param allowEmptyValue boolean
```
Comment (ChristopheCVB) [inline Library/src/main/java/com/christophecvb/touchportal/TouchPortalPlugin.java:649]: Indent this

File: Library/src/test/resources/TouchPortalPluginTest/plugin.config [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #TouchPortalPluginTest
-#Sat May 22 00:00:40 CEST 2021
+#Thu Jun 03 17:37:57 CEST 2021
```
Comment (ChristopheCVB) [inline Library/src/test/resources/TouchPortalPluginTest/plugin.config:2]: In the future, there is no need to commit this file

**Review by ChristopheCVB — CHANGES_REQUESTED** [review]
Just fix the Javadoc indentation and we're fine. Good one 👍

File: Library/src/test/resources/TouchPortalPluginTest/plugin.config (hunk 2 above)
Comment (Pjiesco) [reply Library/src/test/resources/TouchPortalPluginTest/plugin.config:2]: yeah happend accidentally

**Review by ChristopheCVB — APPROVED** [review]

**Pull request merged**